fetch.bs

```diff
@@ -4509,6 +4509,8 @@ steps:
    "<code>http</code>"
    <li><var>request</var>'s <a for=request>current URL</a>'s <a for=url>host</a> is a
    <a for=/>domain</a>
+   <li><var>request</var>'s <a for=request>current URL</a>'s <a for=url>host</a>'s
+   <a for=host>public suffix</a> is not " localhost " or " localhost. "
```

Comment:
Please use the same syntax we use in "resolve an origin". I guess in due course we are gonna need an abstraction for this.

Reply:
I've made an update to add the <code> tags around the localhost hostnames as they were in ResolveAnOrigin. Is that what you were asking for?

"<code>localhost</code>" or "<code>localhost.</code>"